resources/views/waitingrooms/show.blade.php

```diff
@@ -10,8 +10,7 @@
     <div class="container">
         <div class="wrapper">
             <div class="container__block">
-                <p>Campaign ID: {{ $room->campaign_id }}</p>
```

Comment:
@angaither ok cool. Also, i think ideally we should show more user friendly info like Campaign title or something, but maybe we can do that in a different issue. I don't want to add too much to this.